main.py

```diff
@@ -1,70 +1,93 @@
-## Mad libs generator
+#!/usr/bin/env python3
+# Mad libs generator
 
 import time
 
-tps = ['year', 'minute', 'second', 'hour', 'decade', 'week', 'day', 'month', 'decade', 'centur', 'eons', 'forever', 'millen']
+from helpers import *
+
 
 def start():
-    print('welcome to the testing mad libs')
+    print('Welcome to the testing mad libs')
     time.sleep(1)
     print("This is going to be the first instance I test to see what I am capable of so be kind")
     time.sleep(1)
     print("So lets start with the basics...")
     time.sleep(1)
-    name: str=input("What is your name? | ")
-    print("Nice to meet you, "+name.title()+". It's pretty fun to work together like this, isn't it?")
+
+    name: str = input("What is your name? | ")
+    print("Nice to meet you, " + name.title() + ". It's pretty fun to work together like this, isn't it?")
     time.sleep(1)
-    name = name.title() ## why wont this work?
+    name = name.title()
+
     print("Anyway, onto the mad lib")
     time.sleep(1)
-    friend_name: str=input("Name a person you know | ")
-    friend_relationship: str=input("Aaaand this person is what to you? Your... | ")
-    friend_gender= "them"
+    friend_name: str = input("Name a person you know | ")
+    friend_relationship: str = input("Aaaand this person is what to you? Your... | ")
+    friend_relationship = strip_identifiers(friend_relationship).title()
+
+    friend_gender = "them"
     if "boy" in friend_relationship.lower():
         friend_gender = 'him'
     elif "girl" in friend_relationship.lower():
         friend_gender = 'her'
-    keep_swimming(name, friend_name, friend_relationship, friend_gender)
```

Comment:
Rather than have the functions call the next one right after the other, have it return the info it needs then pass that into the next function in an overall main function. 

main functions are used for program flow, and call sub functions.